### DISCUSSION TITLE
token related

### DISCUSSION THREAD

**Discussion by Ismailbukola** [blockscout/blockscout#8733] [Q&A]
Please i'm i supposed to transfer the token into the wallet before mainnet after clicking the explorer alpha and how do i go about it

**Comment by Mojmir-R** [comment]
You must contact directly the team which runs the chain where you are trying to achieve this. This is the wrong place for such question, no one here can help.

**Comment by Togacitygrinds** [comment]
I transferred USDT on Mode to an EVM wallet. it says it was successful but the usdt hasn't been reflected yet after 48hrs. What could be the issue?
This is the txd;  0x98520c7d3393c88c2c284050cf3980f5a92e57ee2580c0e33eec6eb5d3e2e754

**Reply by ekawibisono** [reply]
this is blockscout github, please contact your explorer. thanks
